src/saveload/afterload.cpp

```diff
@@ -3190,6 +3189,15 @@ bool AfterLoadGame()
 			}
 		}
 
+		if (IsSavegameVersionBeforeOrAt(SLV_MULTITRACK_LEVEL_CROSSINGS)) {
```

Comment:
Sure you want this savegame version? Last is (since not long) now `SLV_LINKGRAPH_EDGES` (with a wrong comment).

Reply:
Technically it should indeed have been `SLV_LINKGRAPH_EDGES`, but given the speed the two savegame versions after `SLV_MULTITRACK_LEVEL_CROSSINGS` got added to master, I doubt there are many (if any) savegames that are made between NewGRF road stops got added and this was merged. So, in the grand scheme of things it does not really matter.
I fixed the comment with #10471 by just squashing that tiny change into the same savegame version. Similarly, technically not the right thing to do, but essentially harmless.